src/components/pages/ProgressCharts/ProgressCharts.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
This component might also need to be placed in the `common` folder since it is not a standalone page

Reply:
I moved this component to common and renamed it to NewProgressCharts since there was an older one on there already